### PR TITLE
Delete artifacts that don't expire for some reasons

### PR DESCRIPTION
**Pull request by alberto98fx**
Even tough we defined 5 days as retention day the artifacts are not expiring, since we are calling the Docker actions to handle them they also should be responsible for deletion

### CODE REVIEW

File: .github/workflows/docker.yaml [hunk 1]
```diff
@@ -92,9 +92,33 @@ jobs:
         name: Build with Kaniko
         with:
           push: true
-          build-args: "${{ inputs.BUILD_ARGS }}"
```
Comment (alberto98fx) [inline .github/workflows/docker.yaml:95]: Unknown char here

**Review by mathaeus — COMMENTED** [review]
Should we not first try decreasing it to an hour as discussed in the daily? Or did I misunderstand it?

**Comment by alberto98fx** [comment]
> Should we not first try decreasing it to an hour as discussed in the daily? Or did I misunderstand it?

Can't do it, the action only support "days" https://github.com/actions/upload-artifact

```
  - name: Upload Artifact
    uses: actions/upload-artifact@v3
    with:
      name: my-artifact
      path: my_file.txt
      retention-days: 5
```

**Comment by alberto98fx** [comment]
@mathaeus "The retention period must be between 1 and 90 inclusive. For more information see [artifact and log retention policies](https://docs.github.com/en/free-pro-team@latest/actions/reference/usage-limits-billing-and-administration#artifact-and-log-retention-policy)."

**Comment by mathaeus** [comment]
I see, 1 day then?

**Comment by alberto98fx** [comment]
I thought about that but then I also thought why keeping data that is not used the moment after we finish the action as you pointed out 🤔 so I went ahead and made the fix. 

Do you prefer the 1 day approach? 
Keep in mind tho that web-blog is around 250Mb per build: 
If they do 5 build per day (include tags, dependabot, PRs, merge to master, QA deployments) that's 1.2GB only for 1 day for web-blog, if they expire the next day we paid 1.2GB. 
We have included 2GB per month so we are out of budget on day 2 with web-blog.
Or maybe I misunderstood the billing here which is also very possible 😅

**Review by mathaeus — APPROVED** [review]
I see, thanks for explaining @alberto98fx. Then let's try this.

1. The way how to delete the artifacts, is that code that you come up with yourself or is there a guide how to do that (Just asking to learn more about GHA).
2. Can we DRY the code? From what I can see it's 3 times exactly the same.

**Pull request merged**